x/feemarket/spec/01_concepts.md

```diff
@@ -0,0 +1,20 @@
+<!--
+order: 1
+-->
+
+# Concepts
```

Comment:
Add a section on the `EIP 1559 Fee Market` to give context. I pulled up some of my previous notes, maybe it's helpful: 

- Fixed-per-block network fee (`baseFee`) that is burned
  - `baseFee` per gas changes according to a formula which is a function of gas used in parent block and gas target (block gas limit divided by elasticity multiplier) of parent block.
  - increases when blocks are above the gas target
  - decreases when blocks are below the gas target
- Dynamically expands/contracts block sizes to deal with transient congestion.
- Transactions specify a maximum fee per gas they are willing to pay total (aka: max fee), which covers both the priority fee and the block's network fee per gas (aka: base fee)